app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx

```diff
@@ -0,0 +1,106 @@
+
+export default function ContactTable({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts = [],
+}) {
+  const handleMasterSelectionChange = () => {
+    const contacIds = contacts.map(contact => contact.id);
+    if (selectedContactIds.length === contacIds.length) {
+      setSelectedContactIds([]);
+    } else {
+      setSelectedContactIds(contacIds);
+    }
+  };
+
+  const handleItemSelectionChange = (event, contact) => {
+    event.stopPropagation();
+    const index = selectedContactIds.indexOf(contact.id);
+    if (index > -1) {
+      setSelectedContactIds([
+        ...selectedContactIds.slice(0, index),
+        ...selectedContactIds.slice(index + 1),
+      ]);
+    } else {
+      setSelectedContactIds([...selectedContactIds, contact.id]);
+    }
+  };
+
+  return (
+    <>
+      <div className="w-full p-10">
+        <table className="nui-table nui-table--checkbox nui-table--actions nui-table--hover nui-table--avatar">
+          <thead>
+            <tr>
+              <th>
+                <Checkbox
+                  checked={selectedContactIds.length === contacts.length}
+                  onChange={() => null}
+                  onClick={handleMasterSelectionChange}
+                />
+              </th>
+              <th className="text-left text-gray-400">Name</th>
+              <th className="text-left text-gray-400">Email</th>
+              <th className="text-center text-gray-400">Department</th>
+              <th className="text-center text-gray-400">Contact Number</th>
+              <th className="text-center text-gray-400">Add To Basecamp</th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {contacts.map(contact => (
+              <tr key={contact.id}>
+                <td>
+                  <Checkbox
+                    checked={selectedContactIds.includes(contact.id)}
+                    onChange={() => null}
+                    onClick={event => handleItemSelectionChange(event, contact)}
+                  />
+                </td>
+                <td>
+                  <div className="flex flex-row items-center space-x-3 text-gray-900">
+                    <Avatar size={36} contact={{ name: contact.name }} />
+                    <span>{contact.name}</span>
+                  </div>
+                </td>
+                <td className="max-w-xs truncate">{contact.email}</td>
+                <td className="text-center">{contact.department}</td>
+                <td className="text-center">{contact.contact_number}</td>
+                <td>
+                  <div className="flex flex-row justify-between">
+                    <div className="w-full"></div>
+                    <Checkbox
+                      onChange={() => null}
+                      name="add_to_basecamp"
+                      checked={contact.add_to_basecamp}
+                    />
+                    <div className="w-full"></div>
+                  </div>
```

Comment:
Please update neetoUI to the latest version. 
Then you can simply center align checkbox by wrapping it inside a `div` with  classnames `flex flex-row justify-center`

Reply:
After installing lates  @bigbinary/neetoui": "^1.20.29 and building I'm getting an error with file-loader.

./app/assets/images/EmptyNotesList.svg
Module build failed (from ./node_modules/file-loader/index.js):
TypeError: Cannot read property 'fileLoader' of undefined
    at Object.module.exports (/var/app/node_modules/file-loader/index.js:14:28)